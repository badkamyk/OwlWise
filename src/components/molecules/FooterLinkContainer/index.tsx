import FooterLink from "../../atoms/FooterLink";

export default function FooterLinkContainer({ href }: { href: string }) {
	return (
		<div className="px-5 py-2">
			<FooterLink href={href} />
		</div>
	);
}
