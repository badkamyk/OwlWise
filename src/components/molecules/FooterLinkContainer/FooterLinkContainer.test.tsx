import { render, screen } from "@testing-library/react";
import { expect, it } from "vitest";
import FooterLinkContainer from "./index";

it("FooterLinkContainer", () => {
	const { container } = render(<FooterLinkContainer href={"Home"} />);
	expect(container.getElementsByClassName("px-5 py-2")).toBeTruthy();
	expect(screen.getByRole("link")).toBeTruthy();
	expect(screen.getByRole("link")).toHaveProperty("href", "/");
	expect(screen.getByText("Home")).toBeTruthy();
	expect(container).toMatchSnapshot();
});
