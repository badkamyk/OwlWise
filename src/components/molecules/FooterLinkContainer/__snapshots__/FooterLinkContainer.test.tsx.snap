// Vitest Snapshot v1

exports[`FooterLinkContainer 1`] = `
<div>
  <div
    class="px-5 py-2"
  >
    <a
      class="text-base leading-6 text-gray-100 hover:text-gray-400"
      href="/"
    >
      Home
    </a>
  </div>
</div>
`;

exports[`renders FooterLinkContainer with props correctly 1`] = `
<div>
  <div
    class="px-5 py-2"
  >
    <a
      class="text-base leading-6 text-gray-100 hover:text-gray-400"
      href="/"
    >
      Home
    </a>
  </div>
</div>
`;
